ccache.c

```diff
@@ -2441,6 +2441,31 @@ cc_process_args(struct args *args, struct args **preprocessor_args,
 			free(option);
 			continue;
 		}
+                if (str_startswith(argv[i], "-B")) {
```

Comment:
Please use tabs to indent the code, see HACKING.txt. Tip: You can use `make uncrustify` to make the code use ccache's code conventions.